src/blog/2025-06-01-oxlint-stable.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Oxlint 1.0 Stable
+outline: deep
+authors:
+  - cameron
+---
+
+<AppBlogPostHeader />
+
+Oxlint 1.0 is fully stable. After eighteen months of dedicated development and hundreds of contributions from the community, Oxlint 1.0 is here! We now have a dedicated full-time maintainer (me) and a growing core team working on maintaining and improving the linter.
+
+If you haven’t already, we encourage you to try Oxlint in your codebase. If you run into any issues or have feedback, please reach out - we’re actively listening and ready to respond.
+
+Oxlint a Rust-powered linter for JavaScript and TypeScript that ships with 500+ built-in rules and is designed to be fast and simple to adopt.
+
+## Quick Start
+
+Getting started with Oxlint is straightforward. Run it in your codebase using your preferred package manager:
+
+**Run it, no config required.**
+
+::: code-group
+
+```sh [npm]
+$ npx oxlint@latest
+```
+
+```sh [pnpm]
+$ pnpm dlx oxlint@latest
+```
+
+```sh [yarn]
+$ yarn dlx oxlint@latest
+```
+
+```sh [bun]
+$ bunx oxlint@latest
+```
+
+```sh [deno]
+$ deno run npm:oxlint@latest
+```
+
+:::
+
+For small or new projects, Oxlint can be run without any configuration, using a default set of rules that catch code that is clearly incorrect or unnecessary.
+
+For larger projects or projects that require more customization, Oxlint is configurable via an `.oxlintrc.json` file. This configuration format is based on ESLint v8’s flat config, making migration easy and familiar. Each source file is linted with the nearest applicable configuration, and you can use overrides to target specific glob patterns. You can also extend shared configs to keep teams consistent.
+
+For projects already using ESLint, [oxlint-migrate](https://github.com/oxc-project/oxlint-migrate) can be used to migrate an existing ESLint flat-config file to Oxlint. Additionally, eslint-plugin-oxlint can disable overlapping ESLint rules while both linters are used together. It’s recommended to run `oxlint && eslint` to benefit from Oxlint's faster feedback cycle.
+
+For more detailed instructions on how to use Oxlint and integrate it with your project or editor, check out the [installation guide](/docs/guide/usage/linter).
+
+## Highlights
+
+### Comprehensive Rule Coverage
+
+Oxlint includes over 500 rules from various sources:
+
+- Complete ESLint rule set, including TypeScript-specific rules from typescript-eslint (Excluding type checked rules.)
```

Comment:
For ESLint, we are still missing some rules

> 2/61 recommended rules are remaining as TODO
> 50/137 not recommended rules are remaining as TODO

I think we excluded the stylistic rules already from it. Maybe small notice :)?